Doc/whatsnew/3.13.rst

```diff
@@ -172,6 +172,297 @@ Deprecated
   Replace ``ctypes.SetPointerType(item_type, size)`` with ``item_type * size``.
   (Contributed by Victor Stinner in :gh:`105733`.)
 
+Pending Removal in Python 3.14
+------------------------------
+
+* :mod:`argparse`: The *type*, *choices*, and *metavar* parameters
```

Comment:
I like the practice of putting the module name at the start, and listing them alphabetically, makes it easier to find something.

Let's also alphabetise the main "Deprecated" section. (We could add a hidden comment at the top of the sections saying to add new ones alphabetically - see top of whole file for a hidden `.. Rules for maintenance:` comment).

Reply:
Please go ahead if you want to make a similar change in other sections :-)